src/swarm/neo/node/Connection.d

```diff
@@ -256,8 +256,6 @@ class Connection: ConnectionBase
     override protected void notifyShutdown ( RequestId id, Exception e )
```

Comment:
re: commit message - perhaps it's worth saying why there's there a running request fiber while the send loop's fiber is also running?